helm/botkube/e2e-test-values.yaml

```diff
@@ -34,12 +41,6 @@ sources:
             - create
             - update
             - delete
-        - name: v1/pods
-          namespaces:
-            include:
-              - botkube
-          events:
-            - create
```

Comment:
Not needed anymore, as enabled recommendations will also send a notification 🙂 In that way we're testing the changes from this PR.